content/services/planning.md

```diff
@@ -193,8 +189,21 @@ take the leap into creating your web project!
   ) }}
 {% endcall %}
 
+
 {{ embed.icon_block(planning2) }}
 
+{% call embed.figure(
+  caption='Sample Object Map of Mighty Networks.'
```

Comment:
This looks great. What is Mighty Networks? Does that need some context for others?